source/_integrations/solaredge.markdown

```diff
@@ -36,7 +36,10 @@ In case you would like to convert the values for example to kWh instead of the d
 # Example configuration.yaml entry for template platform
 template:
   - sensor:
-    - name: solaredge_energy_this_year_template:
+    - name: solaredge_energy_this_year_template
+      unit_of_measurement: kWh
+      state_class: total_increasing
+      device_class: energy
```

Comment:
I don't follow the reasoning of this PR, why would that matter? You can just use `sensor.solaredge_energy_this_year` directly with the energy dashboard. Even better, use the provided lifetime sensor.

Reply:
My understanding was wrong. The first time I attempted to add `sensor.solaredge_lifetime_energy` i got an error in the GUI which I wrongly assumed was because the energy dash was expecting kWh, and `sensor.solaredge_lifetime_energy` was in Wh. I'll revise the request to just resolve the simple typo in the template example.